apps/server/src/shared/domain/service/course-copy.service.ts

```diff
@@ -40,12 +40,12 @@ export class CourseCopyService {
 				type: CopyElementType.TIME_GROUP,
 				status: CopyStatusEnum.NOT_DOING,
 			},
-			{
-				type: CopyElementType.COURSEGROUP_GROUP,
-				status: CopyStatusEnum.NOT_IMPLEMENTED,
-			},
 		];
 
+		const courseGroupsExist = params.originalCourse.getCourseLinkedCourseGroups().length > 0;
+		if (courseGroupsExist)
```

Comment:
even is linter not complaining, is a good idea to have `{ }` in the if statement, when it takes more than 1 line

Reply:
sorry totally missed that, you are right :)